data/mods/MMA/modinfo.json

```diff
@@ -3,10 +3,9 @@
     "type": "MOD_INFO",
     "ident": "MMA",
     "name": "Mythical Martial Arts",
-    "authors": [ "Photoloss" ],
+    "authors": [ "Hymore246" ],
```

Comment:
What you actually want here is the "maintainers" field

Reply:
D'oh! I'll fix that.